DS13/code/modules/overmap/events.dm

```diff
@@ -351,4 +351,117 @@ GLOBAL_LIST_INIT(overmap_event_spawns, list())
 		warbird.target = target
 		warbird.linked_event = src
 		elements += warbird
-*/
\ No newline at end of file
+*/
+
+
+
+
+//----------------------
+//ITEM DELIVERY AKA SPACE UPS
+//----------------------
+
+//LANDMARK
+
+GLOBAL_DATUM(sealedcratespawn,/obj/effect/landmark/sealedcrate)
```

Comment:
This isnt a datum?

Reply:
It is,datum is the parent type for all objects